pom.xml

```diff
@@ -127,7 +127,7 @@
     <apache.curator.version>4.2.0</apache.curator.version>
     <aws.amazonaws.version>1.11.215</aws.amazonaws.version>
     <build.path>build</build.path>
-    <copycat.version>1.2.15</copycat.version>
+    <catalyst.version>1.2.1</catalyst.version>
```

Comment:
was it intentional to switch from version 1.2.15?

Reply:
oh i didn't see this one
for catalyst the lastest version is 1.2.1
copycat 1.2.15 uses catalyst 1.2.1